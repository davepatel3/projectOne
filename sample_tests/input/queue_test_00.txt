float
9
2
2
gf
ef 19.8267
ef 81.7756
pb
db
pb
gc
gb
df
pf
qt
