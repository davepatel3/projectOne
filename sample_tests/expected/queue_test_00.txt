cyclic_double_queue is empty!
81.7756
cyclic_double_queue is empty!
9
cyclic_double_queue is empty!
cyclic_double_queue is empty!
cyclic_double_queue is empty!
